index.html

```diff
@@ -11,31 +11,37 @@
   </head>
   <body class="text-center container">
     <main>
-        <h1 class="banner p-3 text-primary">TextEditor</h1>
-        <div class="row">
-          <div class="col">
-            <button id="bold" type="button"  onclick="makeBold(this)" class="btn btn-light">Bold</button>
-            <button id="italic" type="button"  onclick="makeItalic(this)" class="btn btn-light">Italic</button>
-            <button id="underline" type="button"  onclick="makeUnderline(this)" class="btn btn-light">Underline</button>
-            <button id="left-align" type="button" onclick="alignText(this, 'left')" class="align btn btn-light"><i class="fas fa-align-left"></i></button>
-            <button id="center-align" type="button" onclick="alignText(this, 'center')" class="align btn btn-light"><i class="fas fa-align-center"></i></button>
-            <button id="right-align" type="button" onclick="alignText(this, 'right')" class="align btn btn-light"><i class="fas fa-align-right"></i></button>
-          </div>
+      <h1 class="banner p-3 text-primary">TextEditor</h1>
+      <div class="row">
+        <div class="col buttonsInside">
+          <input name="ColorPickerId" type="color" id="ColorPickerName" />
+          <button id="colorChanger" class="btn btn-light" onclick="changeColor()">Change Colour</button>
```

Comment:
- [ ] Remove the button and try to implement change of color while changing the input. Here the text changes color simultaneously with the input.